packages/docs-mintlify/homepage.css

```
@@ -0,0 +1,224 @@
+  user-select: none;
+}
+
+.homepage-language-dropdown-trigger::-webkit-details-marker {
+  display: none;
+}
+
+.homepage-language-dropdown-trigger::after {
+  content: '';
+  width: 0.4rem;
+  height: 0.4rem;
+  border-right: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)